jobs/sync-stream-metadata.Jenkinsfile

```diff
@@ -17,8 +17,7 @@ cosaPod(configMaps: ["fedora-messaging-cfg"], secrets: ["fedora-messaging-coreos
         cmd = """
             aws s3 sync --acl public-read --cache-control 'max-age=60' \
                 --exclude '*' --include 'streams/*' --include 'updates/*' \
-                    ./ s3://fcos-builds
-        """
+                    ./ s3://fcos-builds"""
```

Comment:
I could see someone mistakenly putting back the `"""` on the next line. Any way we could trim the newline in the `def dry_run = shwrapCapture("${cmd} --dryrun").trim()` instead?

Reply:
I changed this to just explicitly `trim()` the `cmd` after defining it.